specification/vmware/resource-manager/Microsoft.AVS/preview/2019-08-09-preview/vmware.json

```diff
@@ -0,0 +1,1277 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure VMware Solution API",
+    "description": "Azure VMware Solution API",
+    "version": "2019-08-09-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "tags": [
+    {
+      "name": "PrivateClouds",
+      "description": ""
+    },
+    {
+      "name": "Clusters",
+      "description": ""
+    },
+    {
+      "name": "Operations",
+      "description": ""
+    },
+    {
+      "name": "Quotas",
+      "description": ""
+    }
+  ],
+  "parameters": {
+    "subscriptionId": {
```

Comment:
For your next version, please remove these common parameters and definitions and reference the standard ones under commont-type\resource-manager\v1\types.json. Including ApiError in all default section.

Reply:
This is already changed in the next API version.